packages/x-components/src/x-modules/search/store/module.ts

```diff
@@ -103,6 +107,9 @@ export const searchXStoreModule: SearchXStoreModule = {
     },
     setRedirections(state, redirections) {
       state.redirections = redirections;
+    },
+    setQueryTagging(state, queryTagging) {
+      state.queryTagging = queryTagging;
```

Comment:
As this is an an object I would rather use `Vue.set()` as the docs says

https://vuex.vuejs.org/guide/mutations.html#mutations-follow-vue-s-reactivity-rules

Am I missing something here? Because is working perfectly.

Reply:
We're adding new properties to an object, as is empty by default. So for me is exactly what the docs says.